rustls/src/suites.rs

```diff
@@ -210,28 +210,52 @@ pub(crate) fn reduce_given_kx_groups(
             _ => g.key_exchange_algorithm() == Some(KeyExchangeAlgorithm::DHE),
         }
     }
-    // https://datatracker.ietf.org/doc/html/rfc7919#section-4 (paragraph 1)
-    let no_known_ffdhe_groups = if let Some(groups_ext) = groups_ext {
-        let mut ext_has_ffdhe_groups = false;
-        let mut ext_has_known_ffdhe_groups = false;
-        for g in groups_ext
-            .iter()
-            .filter(|g| is_ffdhe(g))
-        {
-            ext_has_ffdhe_groups = true;
+
+    let mut ecdhe_kx_ok = false;
```

Comment:
Please squash this commit into the earlier one that introduced these code, in order to make it easier to review.

Reply:
Done!